adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/Adapter.java

```diff
@@ -32,6 +38,7 @@ public Adapter(Importer importer, Interpreter interpreter, DataBlobRepository da
             logger.debug("Fetched: {}", raw);
             JsonNode result = interpreter.interpret(raw, config.formatConfig.parameters);
             DataBlob blob = blobRepository.save(new DataBlob(result.toString()));
+            rabbitTemplate.convertAndSend(RabbitConfiguration.DATA_IMPORT_QUEUE, blob.getMetaData().getLocation());
```

Comment:
Can we send an object with `pipelineId` and `dataLocation` (or `data` as fat event)?

Reply:
Yes, we should do that. But let's leave that for another pull request. This one was meant to provide just the minimal structure (and a working example as part of that)

Reply:
ok!